pages/Components/error.js

```diff
@@ -0,0 +1,27 @@
+//create a error component when user does not exist 
+
+
+
+class ErrorBoundary extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = { hasError: false };
```

Comment:
this.props = { errorMessage };

Refer to index.js for SearchPage where there is a line for const[error, setError] = useState(). The error message is set in the function search on this page and stored in the components state, this will be passed to your component as a prop.

Reply:
So you can remove this.state hasError and replace with props